modules/reactor/doc/content/source/meshgenerators/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator.md

```diff
@@ -0,0 +1,40 @@
+# CartesianConcentricCircleAdaptiveBoundaryMeshGenerator
+
+!syntax description /Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator
+
+## Overview
+
+The `CartesianConcentricCircleAdaptiveBoundaryMeshGenerator` object generates a 2D mesh for concentric circles with a square enclosure with or without external ducts. It is similar to [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) and the majority of the parameters and functionalities of this object are described in the documentation of [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md). The major unique feature of this object is that the [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/num_sectors_per_side) parameter will be overridden to enable stitching to another square mesh (provided via [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/inputs) to this object). This object therefore creates square meshes that can be directly stitched with square meshes generated by [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) and [`PatternedCartesianMeshGenerator`](/PatternedCartesianMeshGenerator.md). This object only works for sqaures instead of general polygons.
+
+### More Information
+
+In order to adapt the external boundary to match input meshes (typically generated by [`PatternedCartesianMeshGenerator`](/PatternedCartesianMeshGenerator.md)), two extra input parameters must be provided:
+
+- [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/sides_to_adapt): a vector parameter that can include up to six elements which lists the reference square side numbers to adaptively mesh. These entries must be non-repeating integers from 0 to 3 (see [Figure 1](#schematic)). The square sides are indexed counterclockwise starting from the top side.
```

Comment:
I dont understand why 6 elements / inputs? you only have 4 sides to adapt?

Reply:
Should be four. Failed to capture this when copying and modifying the hexagonal MG's documentation.